src/notation/internal/notationinteraction.cpp

```diff
@@ -2032,7 +2032,7 @@ void NotationInteraction::applyDropPaletteElement(mu::engraving::Score* score, m
             }
         }
 
-        if (el && el->hasVoiceApplicationProperties()) {
+        if (el && el->hasVoiceApplicationProperties() && !el->discardDefaultVoiceApplication()) {
```

Comment:
Maybe we should just check the el type here instead of adding this virtual method?